src/main/java/com/mraof/minestuck/client/gui/AnthvilScreen.java

```diff
@@ -0,0 +1,123 @@
+package com.mraof.minestuck.client.gui;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mraof.minestuck.Minestuck;
+import com.mraof.minestuck.alchemy.*;
+import com.mraof.minestuck.alchemy.recipe.GristCostRecipe;
+import com.mraof.minestuck.blockentity.machine.AnthvilBlockEntity;
+import com.mraof.minestuck.client.util.GuiUtil;
+import com.mraof.minestuck.inventory.AnthvilMenu;
+import com.mraof.minestuck.network.AnthvilPacket;
+import com.mraof.minestuck.network.MSPacketHandler;
+import com.mraof.minestuck.network.MachinePacket;
+import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.client.gui.widget.ExtendedButton;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+public class AnthvilScreen extends MachineScreen<AnthvilMenu>
+{
+	private static final ResourceLocation BACKGROUND_TEXTURE = new ResourceLocation(Minestuck.MOD_ID, "textures/gui/anthvil.png");
+	private static final ResourceLocation FUEL_STATUS = new ResourceLocation(Minestuck.MOD_ID, "textures/gui/progress/uranium_level.png");
+	
+	private static final int FUEL_STATUS_X = 133;
+	private static final int FUEL_STATUS_Y = 7;
+	private static final int STATUS_WIDTH = 35;
+	private static final int STATUS_HEIGHT = 39;
+	private static final int BUTTON_X = 12;
+	private static final int BUTTON_Y = 38;
+	private final AnthvilBlockEntity anthvilBE;
```

Comment:
Looks like this is only here for level access. I suggest removing it and getting the level from the minecraft instance attached to the screen instead (`MiniAlchemiterScreen`).